client/src/components/Pets/PetList.tsx

```diff
@@ -60,9 +60,7 @@ const PetList = ({ user, profile }: Props): JSX.Element => {
         <Grid key={p.name} item xs={6}>
           <Box className={classes.petCard}>
             <Box className={classes.petPhotoBox}>
-              {p.petPhoto instanceof File ? (
-                ''
-              ) : (
+              {!(p.petPhoto instanceof File) && (
```

Comment:
This should double a bang `!!(p.petPhoto ...` Refer to my previous comment :smile: